wai-aria/tools/atta-example.py

```diff
@@ -0,0 +1,289 @@
+# atta-example
```

Comment:
Question: Do we want to include this? Given the amount of work required for a fully-functional ATTA, this might be a hello-world file without further guidance. Were it me, I would be tempted to leave it out.